### PR TITLE
Hello-rails-react with react-rails gem

### PR DESCRIPTION
**Pull request by ZeenatLawal**
In this PR, I:

- Created a new Rails app called 'hello-rails-react'.
- Set up the project with webpacker the react-rails gems.
- Set up Postgres database.
- Created a table for storing messages and created 5 different greetings.
- Created an API endpoint that selects a random greeting from the table.
- Created a static view that is the root of your app.
- Created the App component with react-router and rendered it in the static view.
- Created the Greeting component that displays a greeting. Added as a route in the App component.
- Created a store, an action and a reducer that connects to the API endpoint to get the random greeting.
- Displayed the random greeting in the Greeting component.

### CODE REVIEW

File: app/controllers/api/greetings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Api::GreetingsController < ApplicationController
+  def index
+    @greeting = Greeting.order('RANDOM()').first
+    render json: @greeting
+  end
+end
```
Comment (Emmyn5600) [inline app/controllers/api/greetings_controller.rb:6]: - [ ] rendering the random greeting doesn't work, please make sure that a user sees the random greeting once he or she visited the correct routes as well as get the random greeting.
data

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi, @ZeenatLawal 

Good job so far! 👏🏼 

Great work on making the project requirements:clap:
You've done well implementing the project requirements, but there are still some issues which aren't been addressed yet 
 
## STATUS: CHANGES REQUESTED :recycle:


### Changes requested.

- [ ] Check the comment under the review.

 #### Optional suggestions
 
 - N/A
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: app/controllers/api/greetings_controller.rb (hunk 1 above)
Comment (ZeenatLawal) [reply app/controllers/api/greetings_controller.rb:6]: Hello @Emmyn5600, the random greeting shows directly on the root page, you don't need to go to another path. Kindly have another look

Comment (Emmyn5600) [reply app/controllers/api/greetings_controller.rb:6]: @ZeenatLawal it doesn’t work when  I visit the root path as well

Comment (Emmyn5600) [reply app/controllers/api/greetings_controller.rb:6]: data1

**Review by Emmyn5600 — APPROVED** [review]
Hi @ZeenatLawal 

Good job👏🏼

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the project requirements

 #### Optional suggestions
 -  N/A.
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**